src/OnScreen/BackgroundWindow.vala

```diff
@@ -387,7 +387,7 @@ namespace Komorebi.OnScreen {
 
 			wallpaperActor.set_content(wallpaperImage);
 
-			wallpaperPixbuf = new Gdk.Pixbuf.from_file_at_scale(@"$package_datadir/$wallpaperName/wallpaper.jpg",
+			wallpaperPixbuf = new Gdk.Pixbuf.from_file_at_scale(@"$wallpaperPath/wallpaper.jpg",
 																scaleWidth, scaleHeight, false);
```

Comment:
I know that you didn't cause this, but I'll get you to fix up the indentation here.

Reply:
Totally right :stuck_out_tongue: fixed